### PR TITLE
CLDR-11157 show emoji code identifiers

### PR DESCRIPTION
**Pull request by btangmu**
-Add method DataSection.getRootFile for performance

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-11157
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-apps/src/org/unicode/cldr/web/DataSection.java [hunk 1]
```diff
@@ -93,7 +93,12 @@
      * The baseline CLDRFile for this DataSection
      */
     public CLDRFile translationHintsFile;
-    
+
+    /**
+     * The CLDRFile for the root locale; use lazy initialization, see getRootFile
+     */
+    private CLDRFile rootFile = null;
```
Comment (macchiati) [inline tools/cldr-apps/src/org/unicode/cldr/web/DataSection.java]: Note for all.
There are three files that must never change while the ST is active:
en.xml
root.xml
and the translationHints file.

So we could have statics for those to save building them for any new instance.

**Comment by btangmu** [comment]
I don't know what to do about "license/cla Expected — Waiting for status to be reported"?

**Comment by macchiati** [comment]
> I don't know what to do about "license/cla Expected — Waiting for status to be reported"?

Should be ok (see also email)

File: tools/cldr-apps/src/org/unicode/cldr/web/DataSection.java (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/src/org/unicode/cldr/web/DataSection.java]: I'll make a ticket for that. Maybe the statics should be for all of ST, not just DataSection?

**Pull request merged**